.github/workflows/test_firedancer.yml

```diff
@@ -0,0 +1,30 @@
+name: Firedancer
+on:
+  workflow_call:
+  workflow_dispatch:
+  pull_request:
+jobs:
+  firedancer-tests:
+    runs-on: private
+    env:
+      CC: gcc
+      EXTRAS: no-solana
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: recursive
+
+      - uses: ./.github/actions/deps
+      - uses: ./.github/actions/hugepages
+        with:
+          count: 128
+
+      - name: build
+        run: |
+          ./contrib/make-j fddev all
+      - name: test firedancer - local cluster
+        run: |
+          sudo prlimit --pid=$$ --nofile=1048576
+          sudo prlimit --pid=$$ --memlock=unlimited
```

Comment:
You don't need these, fddev handles nofile and memlock.

Reply:
done

Reply:
nvm, undid. these are set up for a local cluster we set up for the script. fddev doesnt do this until much later.